DOCS.md

```diff
@@ -42,6 +42,10 @@ drone secret add --image=plugins/docker \
 
 drone secret add --image=plugins/docker \
     octocat/hello-world DOCKER_EMAIL kevin.bacon@mail.com
+
+# Example adding JSON key file for GCR, it reads contents of file google-key.json
+drone secret add --image=plugins/docker \
```

Comment:
agreed. the README is for developers that compile and run the plugin locally. The DOCS.md file is for usage instructions and will auto-generate documentation at readme.drone.io/plugins/docker

let's remove these lines